### DISCUSSION TITLE
Other ideas

### DISCUSSION THREAD

**Discussion by gmelodie** [MLH-Fellowship/prep-project-jan-2022#33] [Ideas]
- Link to react docs
- Clippy-like bot
- How many days of weather report it should display
- [Lottie](https://lottiefiles.com/) animations for items they should bring instead of pictures
- Present data in the form of graphs (humidity, heat maps, etc.)
- Query things like hotels and places to eat
- Audio features (wind sounds if it's windy, rain sounds if it's raining)
- Extensibility: different, reusable components on other files and just import them `App.js`
- Add more weather details (sunrise/set times)
- Create a more compelling design (any good designers there?) -> @Chukwuemeka-Mba + someone else

**Comment by KhudaDad414** [comment]
@MLH-Fellowship/mlh-prep-january-2022 Hey fellows, I have checked and 5/6 current PRs are editing `App.js` (I assume other fellows are doing it as well). I am going to refactor my PR so I don't have to deal with conflicts, I suggest you rethink your approach and if possible, move your logic in functions or something and try your best to add things instead of editing. good luck (we definitely need it 😄)

**Comment by ditsuke** [comment]
How about local news, events, hotel bookings and places to eat?

**Comment by CIPHERTron** [comment]
> How about local news, events, hotel bookings and places to eat?

More like a trip planner? xD

**Comment by ditsuke** [comment]
> > How about local news, events, hotel bookings and places to eat?
> 
> More like a trip planner? xD

An ideas an idea 😛
